src/components/EnlargedPreview/index.js

```diff
@@ -47,37 +50,49 @@ class EnlargedPreview extends Component<Props, State> {
   }
 
   handleImageLoad = () => {
-    if (this.container) {
-      this.container.center()
+    if (this.image) {
+      this.image.center()
     }
   }
 
   toggle = () => this.setState({
     isExpanded: !this.state.isExpanded,
     hasEntered: false,
-  }, () =>
+  }, () => {
     this.setState({ hasEntered: true })
+
+    if (this.previewContainer) {
+      this.previewContainer.focus()
+    }
+  }
   )
 
   render() {
     const { isExpanded, hasEntered } = this.state
-    const { translate, src } = this.props
+    const { translate, src, altTag, enlargedAltTag } = this.props
     return (
       <div
         className={classNames({
           [style.expanded]: isExpanded,
           [style.entered]: hasEntered,
         }, style.container)}
       >
-      {
-        isExpanded &&
-          <Pannable
-            ref={ node => this.container = node }
-            className={style.imageContainer}
-          >
-            <img onLoad={this.handleImageLoad} className={style.image} src={src} />
-          </Pannable>
-      }
+        <div
+          ref={node => this.previewContainer = node}
+          tabIndex={-1}
+          aria-label={isExpanded ? enlargedAltTag : altTag}
+          aria-hidden="true" // hide regular group announcement in order to have label announced on each focus
+        >
+          {
+            isExpanded &&
+              <Pannable
+                ref={ node => this.image = node }
+                className={style.imageContainer}
+              >
+                <img onLoad={this.handleImageLoad} className={style.image} src={src} alt={enlargedAltTag} />
```

Comment:
This alt tag is never read by the screenreader. It's only the parent aria-label that gets announced but there is no mention of the image when you are on the enlarged screen, which I think can be confusing to accessibility users.